chall03/gmartine.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+direction = [1, 0]
+position = [0, 0]
+repeat = 1
+flag  = 1
+
+if (len(sys.argv) == len(sys.argv[1]) + 1):
+    if (sys.argv[1].isalnum() and sys.argv[2].isalnum() and sys.argv[3].isalnum() and len(sys.argv[1]) == len(sys.argv[2]) and len(sys.argv[2]) == len(sys.argv[3])):
```

Comment:
You should think the other way around. and do the error conditions first with an extern function of error handling. It will simplify your code